pallets/thea-staking/src/lib.rs

```diff
@@ -659,6 +844,39 @@ impl<T: Config> Pallet<T> {
 		<StakingData<T>>::remove(session_to_delete, network);
 		log::trace!(target: "runtime::thea::staking", "removing staking data of session {:?} and network {:?}", session_to_delete,network);
 	}
+
+	// making sure we're not exceeding 100% and not below 1%
+	fn moderate_slashing_coeficient() -> u8 {
+		let set = T::ModerateSlashingCoeficient::get();
+		if !(1..=100).contains(&set) {
+			Self::deposit_event(Event::<T>::MisconfiguredCoeficient("Moderate".into()));
+			5
+		} else {
+			set
+		}
+	}
+
+	// making sure we're not exceeding 100% and not below 1%
+	fn severe_slashing_coeficient() -> u8 {
+		let set = T::SevereSlashingCoeficient::get();
+		if !(1..=100).contains(&set) {
+			Self::deposit_event(Event::<T>::MisconfiguredCoeficient("Severe".into()));
+			20
+		} else {
+			set
+		}
+	}
+
+	// making sure we're not exceeding 100% and not below 1%
+	fn threshold_slashing_coeficient() -> u8 {
+		let set = T::SlashingThreshold::get();
+		if !(1..=100).contains(&set) {
+			Self::deposit_event(Event::<T>::MisconfiguredCoeficient("Threshold".into()));
+			60
```

Comment:
Could this value be at least as a constant?

Reply:
done